src/System-Announcements/ClassRenamed.class.st

```diff
@@ -76,3 +76,10 @@ ClassRenamed >> oldName: anObject [
 	
 	oldName := anObject
 ]
+
+{ #category : #accessing }
+ClassRenamed >> packagesAffected [
+	^ (super packageAffected,
+		(self classAffected methods asArray , self classAffected classSide methods asArray
```

Comment:
I think it can be simplified by:
```Smalltalk
^self classAffected packages
```
"aMethod package" in fact also call this code. So it should be correct and optimized version

Reply:
And it takes into account instance and class sides